docs/administrative/readme.md

```diff
@@ -0,0 +1,17 @@
+# Readme
+
+In this folder all information for the functionality of the Knights can be found.
+For the purpose of these documents, please adhere to the following definitions:
+
+**Squire**
+The internal platform the Knights as is active on the site
+
+**admins** 
+Persons with a high form of administrative power in the backend. In rule this should be board and the
```

Comment:
I disagree that the board should be admins. They should have more rights than normal members (or else they can't actually do their administration), but they should not be able to touch certain settings (such as global permissions). Compare this to, say, the mailserver, where the board also does not get admin powers.

Reply:
I agree that the board will pretty much always have access to (part of) the admin panel. Perhaps explicitly mention that the board won't have access to the entire backend?